### ISSUE TITLE
Grid, Replicator breaks GraphQL

### ISSUE THREAD

**Issue by goellner** [statamic/cms#8324]
### Bug description

When using grid or replicator fields in a blueprint it breaks graphql and throws an error

```
[2023-06-19 10:02:13] local.ERROR: Type Sets_ReplicatorField not found.
```

### How to reproduce

Add a grid or replicator field to the user blueprint (or any other blueprint) try to open the graphql explorer.

### Logs

```shell
[2023-06-19 09:58:17] local.ERROR: Type Sets_ReplicatorField not found.
Check that the config array key for the type matches the name attribute in the type's class.
It is required when 'lazyload_types' is enabled {"exception":"[object] (Rebing\\GraphQL\\Exception\\TypeNotFound(code: 0): Type Sets_ReplicatorField not found.
Check that the config array key for the type matches the name attribute in the type's class.
It is required when 'lazyload_types' is enabled at /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/GraphQL.php:251)
[stacktrace]
#0 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/GraphQL.php(227): Rebing\\GraphQL\\GraphQL->getType('Sets_Replicator...', false)
#1 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(353): Rebing\\GraphQL\\GraphQL->type('Sets_Replicator...')
#2 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/GraphQL/Manager.php(36): Illuminate\\Support\\Facades\\Facade::__callStatic('type', Array)
#3 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(353): Statamic\\GraphQL\\Manager->type('Sets_Replicator...')
#4 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Fieldtypes/Replicator.php(253): Illuminate\\Support\\Facades\\Facade::__callStatic('type', Array)
#5 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Fields/Field.php(391): Statamic\\Fieldtypes\\Replicator->toGqlType()
#6 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Collections/HigherOrderCollectionProxy.php(60): Statamic\\Fields\\Field->toGql()
#7 [internal function]: Illuminate\\Support\\HigherOrderCollectionProxy->Illuminate\\Support\\{closure}(Object(Statamic\\Fields\\Field), 'replicator_fiel...')
#8 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Collections/Arr.php(558): array_map(Object(Closure), Array, Array)
#9 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Collections/Collection.php(771): Illuminate\\Support\\Arr::map(Array, Object(Closure))
#10 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Collections/HigherOrderCollectionProxy.php(61): Illuminate\\Support\\Collection->map(Object(Closure))
#11 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Fields/Fields.php(311): Illuminate\\Support\\HigherOrderCollectionProxy->__call('toGql', Array)
#12 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/GraphQL/Types/UserType.php(19): Statamic\\Fields\\Fields->toGql()
#13 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/Type.php(78): Statamic\\GraphQL\\Types\\UserType->fields()
#14 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/Type.php(114): Rebing\\GraphQL\\Support\\Type->getFields()
#15 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Type/Definition/FieldDefinition.php(97): Rebing\\GraphQL\\Support\\Type->Rebing\\GraphQL\\Support\\{closure}()
#16 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Type/Definition/TypeWithFields.php(26): GraphQL\\Type\\Definition\\FieldDefinition::defineFieldMap(Object(GraphQL\\Type\\Definition\\ObjectType), Object(Closure))
#17 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Type/Definition/TypeWithFields.php(61): GraphQL\\Type\\Definition\\TypeWithFields->initializeFields()
#18 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Utils/TypeInfo.php(202): GraphQL\\Type\\Definition\\TypeWithFields->getFields()
#19 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Utils/TypeInfo.php(162): GraphQL\\Utils\\TypeInfo::extractTypes(Object(GraphQL\\Type\\Definition\\ObjectType), Array)
#20 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Utils/TypeInfo.php(222): GraphQL\\Utils\\TypeInfo::extractTypes(Object(GraphQL\\Type\\Definition\\ListOfType), Array)
#21 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Utils/TypeInfo.php(222): GraphQL\\Utils\\TypeInfo::extractTypes(Object(Rebing\\GraphQL\\Support\\PaginationType), Array)
#22 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Type/Schema.php(222): GraphQL\\Utils\\TypeInfo::extractTypes(Object(GraphQL\\Type\\Definition\\ObjectType), Array)
#23 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Type/Schema.php(208): GraphQL\\Type\\Schema->collectAllTypes()
#24 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Type/Introspection.php(240): GraphQL\\Type\\Schema->getTypeMap()
#25 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(623): GraphQL\\Type\\Introspection::GraphQL\\Type\\{closure}(Object(GraphQL\\Type\\Schema), Array, NULL, Object(GraphQL\\Type\\Definition\\ResolveInfo))
#26 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(550): GraphQL\\Executor\\ReferenceExecutor->resolveFieldValueOrError(Object(GraphQL\\Type\\Definition\\FieldDefinition), Object(GraphQL\\Language\\AST\\FieldNode), Object(Closure), Object(GraphQL\\Type\\Schema), Object(GraphQL\\Type\\Definition\\ResolveInfo))
#27 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(1195): GraphQL\\Executor\\ReferenceExecutor->resolveField(Object(GraphQL\\Type\\Definition\\ObjectType), Object(GraphQL\\Type\\Schema), Object(ArrayObject), Array)
#28 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(1145): GraphQL\\Executor\\ReferenceExecutor->executeFields(Object(GraphQL\\Type\\Definition\\ObjectType), Object(GraphQL\\Type\\Schema), Array, Object(ArrayObject))
#29 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(1106): GraphQL\\Executor\\ReferenceExecutor->collectAndExecuteSubfields(Object(GraphQL\\Type\\Definition\\ObjectType), Object(ArrayObject), Array, Object(GraphQL\\Type\\Schema))
#30 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(793): GraphQL\\Executor\\ReferenceExecutor->completeObjectValue(Object(GraphQL\\Type\\Definition\\ObjectType), Object(ArrayObject), Object(GraphQL\\Type\\Definition\\ResolveInfo), Array, Object(GraphQL\\Type\\Schema))
#31 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(741): GraphQL\\Executor\\ReferenceExecutor->completeValue(Object(GraphQL\\Type\\Definition\\ObjectType), Object(ArrayObject), Object(GraphQL\\Type\\Definition\\ResolveInfo), Array, Object(GraphQL\\Type\\Schema))
#32 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(654): GraphQL\\Executor\\ReferenceExecutor->completeValue(Object(GraphQL\\Type\\Definition\\NonNull), Object(ArrayObject), Object(GraphQL\\Type\\Definition\\ResolveInfo), Array, Object(GraphQL\\Type\\Schema))
#33 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(557): GraphQL\\Executor\\ReferenceExecutor->completeValueCatchingError(Object(GraphQL\\Type\\Definition\\NonNull), Object(ArrayObject), Object(GraphQL\\Type\\Definition\\ResolveInfo), Array, Object(GraphQL\\Type\\Schema))
#34 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(1195): GraphQL\\Executor\\ReferenceExecutor->resolveField(Object(GraphQL\\Type\\Definition\\ObjectType), NULL, Object(ArrayObject), Array)
#35 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(264): GraphQL\\Executor\\ReferenceExecutor->executeFields(Object(GraphQL\\Type\\Definition\\ObjectType), NULL, Array, Object(ArrayObject))
#36 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php(215): GraphQL\\Executor\\ReferenceExecutor->executeOperation(Object(GraphQL\\Language\\AST\\OperationDefinitionNode), NULL)
#37 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/Executor/Executor.php(156): GraphQL\\Executor\\ReferenceExecutor->doExecute()
#38 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/GraphQL.php(162): GraphQL\\Executor\\Executor::promiseToExecute(Object(GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter), Object(GraphQL\\Type\\Schema), Object(GraphQL\\Language\\AST\\DocumentNode), NULL, NULL, NULL, NULL, NULL)
#39 /Users/elektro/code/itsasecret/apps/cms/vendor/webonyx/graphql-php/src/GraphQL.php(94): GraphQL\\GraphQL::promiseToExecute(Object(GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter), Object(GraphQL\\Type\\Schema), Object(GraphQL\\Language\\AST\\DocumentNode), NULL, NULL, NULL, NULL, NULL, NULL)
#40 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/GraphqlExecutionMiddleware.php(32): GraphQL\\GraphQL::executeQuery(Object(GraphQL\\Type\\Schema), Object(GraphQL\\Language\\AST\\DocumentNode), NULL, NULL, NULL, NULL, NULL)
#41 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/AbstractExecutionMiddleware.php(34): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\GraphqlExecutionMiddleware->handle('default', Object(GraphQL\\Type\\Schema), Object(Rebing\\GraphQL\\Support\\OperationParams), NULL, NULL, Object(Closure))
#42 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\AbstractExecutionMiddleware->resolve(Array, Object(Closure))
#43 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/AbstractExecutionMiddleware.php(33): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Array)
#44 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/AddAuthUserContextValueMiddleware.php(28): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\AbstractExecutionMiddleware->Rebing\\GraphQL\\Support\\ExecutionMiddleware\\{closure}('default', Object(GraphQL\\Type\\Schema), Object(Rebing\\GraphQL\\Support\\OperationParams), NULL, NULL)
#45 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/AbstractExecutionMiddleware.php(34): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\AddAuthUserContextValueMiddleware->handle('default', Object(GraphQL\\Type\\Schema), Object(Rebing\\GraphQL\\Support\\OperationParams), NULL, NULL, Object(Closure))
#46 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\AbstractExecutionMiddleware->resolve(Array, Object(Closure))
#47 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/AbstractExecutionMiddleware.php(33): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Array)
#48 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/AutomaticPersistedQueriesMiddleware.php(42): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\AbstractExecutionMiddleware->Rebing\\GraphQL\\Support\\ExecutionMiddleware\\{closure}('default', Object(GraphQL\\Type\\Schema), Object(Rebing\\GraphQL\\Support\\OperationParams), NULL, NULL)
#49 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/AbstractExecutionMiddleware.php(34): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\AutomaticPersistedQueriesMiddleware->handle('default', Object(GraphQL\\Type\\Schema), Object(Rebing\\GraphQL\\Support\\OperationParams), NULL, NULL, Object(Closure))
#50 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\AbstractExecutionMiddleware->resolve(Array, Object(Closure))
#51 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/AbstractExecutionMiddleware.php(33): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Array)
#52 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/ValidateOperationParamsMiddleware.php(39): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\AbstractExecutionMiddleware->Rebing\\GraphQL\\Support\\ExecutionMiddleware\\{closure}('default', Object(GraphQL\\Type\\Schema), Object(Rebing\\GraphQL\\Support\\OperationParams), NULL, NULL)
#53 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Support/ExecutionMiddleware/AbstractExecutionMiddleware.php(34): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\ValidateOperationParamsMiddleware->handle('default', Object(GraphQL\\Type\\Schema), Object(Rebing\\GraphQL\\Support\\OperationParams), NULL, NULL, Object(Closure))
#54 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Rebing\\GraphQL\\Support\\ExecutionMiddleware\\AbstractExecutionMiddleware->resolve(Array, Object(Closure))
#55 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Array)
#56 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#57 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/GraphQL.php(158): Illuminate\\Pipeline\\Pipeline->thenReturn()
#58 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/GraphQL.php(141): Rebing\\GraphQL\\GraphQL->executeViaMiddleware(Array, 'default', Object(GraphQL\\Type\\Schema), Object(Rebing\\GraphQL\\Support\\OperationParams), NULL, NULL)
#59 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/GraphQL.php(127): Rebing\\GraphQL\\GraphQL->executeAndReturnResult('default', Object(GraphQL\\Type\\Schema), Object(Rebing\\GraphQL\\Support\\OperationParams), NULL, NULL)
#60 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/GraphQLController.php(43): Rebing\\GraphQL\\GraphQL->execute('default', Object(Rebing\\GraphQL\\Support\\OperationParams))
#61 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/Helpers.php(25): Rebing\\GraphQL\\GraphQLController->Rebing\\GraphQL\\{closure}(Object(GraphQL\\Server\\OperationParams))
#62 /Users/elektro/code/itsasecret/apps/cms/vendor/rebing/graphql-laravel/src/GraphQLController.php(40): Rebing\\GraphQL\\Helpers::applyEach(Object(Closure), Object(GraphQL\\Server\\OperationParams))
#63 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): Rebing\\GraphQL\\GraphQLController->query(Object(Illuminate\\Http\\Request), Object(Laragraph\\Utils\\RequestParser), Object(Illuminate\\Config\\Repository), Object(Rebing\\GraphQL\\GraphQL))
#64 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(43): Illuminate\\Routing\\Controller->callAction('query', Array)
#65 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Routing/Route.php(260): Illuminate\\Routing\\ControllerDispatcher->dispatch(Object(Illuminate\\Routing\\Route), Object(Rebing\\GraphQL\\GraphQLController), 'query')
#66 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Routing/Route.php(205): Illuminate\\Routing\\Route->runController()
#67 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Routing/Router.php(799): Illuminate\\Routing\\Route->run()
#68 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#69 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Http/Middleware/HandleToken.php(13): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#70 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Statamic\\Http\\Middleware\\HandleToken->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#71 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Http/Middleware/RequireStatamicPro.php(17): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#72 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Statamic\\Http\\Middleware\\RequireStatamicPro->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#73 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Http/Middleware/SwapExceptionHandler.php(19): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#74 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Statamic\\Http\\Middleware\\SwapExceptionHandler->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#75 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/GraphQL/Middleware/CacheResponse.php(22): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#76 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Statamic\\GraphQL\\Middleware\\CacheResponse->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#77 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#78 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Routing/Router.php(800): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#79 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Routing/Router.php(777): Illuminate\\Routing\\Router->runRouteWithinStack(Object(Illuminate\\Routing\\Route), Object(Illuminate\\Http\\Request))
#80 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Routing/Router.php(741): Illuminate\\Routing\\Router->runRoute(Object(Illuminate\\Http\\Request), Object(Illuminate\\Routing\\Route))
#81 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Routing/Router.php(730): Illuminate\\Routing\\Router->dispatchToRoute(Object(Illuminate\\Http\\Request))
#82 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(200): Illuminate\\Routing\\Router->dispatch(Object(Illuminate\\Http\\Request))
#83 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}(Object(Illuminate\\Http\\Request))
#84 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Http/Middleware/DisableFloc.php(18): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#85 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Statamic\\Http\\Middleware\\DisableFloc->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#86 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Http/Middleware/CheckMultisite.php(14): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#87 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Statamic\\Http\\Middleware\\CheckMultisite->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#88 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Http/Middleware/CheckComposerJsonScripts.php(14): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#89 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Statamic\\Http\\Middleware\\CheckComposerJsonScripts->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#90 /Users/elektro/code/itsasecret/apps/cms/vendor/statamic/cms/src/Http/Middleware/PoweredByHeader.php(19): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#91 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Statamic\\Http\\Middleware\\PoweredByHeader->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#92 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#93 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#94 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#95 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#96 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#97 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#98 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#99 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#100 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#101 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#102 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Http/Middleware/HandleCors.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#103 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\HandleCors->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#104 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#105 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\TrustProxies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#106 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#107 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(175): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#108 /Users/elektro/code/itsasecret/apps/cms/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(144): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))
#109 /Users/elektro/code/itsasecret/apps/cms/public/index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
#110 /Users/elektro/.composer/vendor/laravel/valet/server.php(110): require('/Users/elektro/...')
#111 {main}
"}
```

### Environment

```yaml
Environment
Application Name: Statamic
Laravel Version: 10.13.5
PHP Version: 8.1.18
Composer Version: 2.5.5
Environment: local
Debug Mode: ENABLED
URL: graphqlshizzle.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 0
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: 4.6.0 PRO
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

None

### Additional details

Tested on 4.3.0 to 4.6.0, can't use grid/replicator on any of those releases.

**Comment by jesseleite** [comment]
@goellner I can't reproduce here. Do you see any differences in how we're querying? 🤔

![CleanShot 2023-06-19 at 13 31 39](https://github.com/statamic/cms/assets/5187394/4434e377-2e4a-4c5f-9857-1fee3b3419de)

**Comment by goellner** [comment]
I created a github repo with a fresh statamic install and the second commit contains everythingto recreate the bug:

https://github.com/goellner/graphql-bug/commit/6ba3dca496fc29d6d3025bda5fc7839ff5a62382

Can't query anything because as soon as I open the graphiql interface and open the explorer the error is in the logs.

**Comment by goellner** [comment]
Narrowed the bug down a bit more. If the grid/replicator is only added to the user blueprint it breaks (same for runway blueprints). Adding a grid field to a `pages` collection blueprint it works. If the grid item is named the same (e.g. `my_grid`) on the page blueprint and the user blueprint it works, because the type gets generated.

**Comment by jesseleite** [comment]
Good find @goellner, thank you. Fixed in #8335 👍
